tests/integration/targets/ios_user/tests/cli/basic.yaml

```diff
@@ -179,6 +179,7 @@
   cisco.ios.ios_user:
     aggregate:
       - name: ciscoios
```

Comment:
This was a user not created for the CI system, this can be removed!

Reply:
Okay!